extensions/git/src/api/api1.ts

```diff
@@ -112,6 +112,10 @@ export class ApiRepository implements Repository {
 		return this.#repository.setConfig(key, value);
 	}
 
+	unsetConfig(key: string): Promise<string> {
+		return this.#repository.unsetConfig(key);
```

Comment:
Last I heard we weren't meant to use # privates yet, I think due to performance concersn?

Reply:
My understanding is that using native privates is one of the suggested options to make sure that we are not leaking data that is meant to be internal. We are using native privates in multiple places in core to make sure that we do not leak data to extensions.